site/public/js/userform.js

```diff
@@ -204,9 +204,11 @@ function autoCompleteOnUserId(user_information) {
 }
 
 function completeUserFormInformation(user) {
-    var form = $("#edit-user-form");
+  var form = $("#edit-user-form");
+
+  $('[name="edit_user"]', form).val("true");
 
-    $('[name="user_numeric_id"]', form).val(user['user_numeric_id']);
+  $('[name="user_numeric_id"]', form).val(user['user_numeric_id']);
```

Comment:
It looks like you removed some white-space here and that messes with the indentation. I think it looks better with the original white-space where everything lines up.